src/FSharp.Markdown/HtmlFormatting.fs

```diff
@@ -49,24 +50,27 @@ type UniqueNameGenerator() =
             name
 
 /// Context passed around while formatting the HTML
-type FormattingContext =
-  { LineBreak : unit -> unit
+type HtmlFormattingContext = {
+    LineBreak : unit -> unit
```

Comment:
Personally I prefer the old indentation, see https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-record-declarations